<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="FloorPanelFrame">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="FloorPanel_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: Black;
 font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Floor Panel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="FloorPanel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EDE9E9
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="startFloor_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: Black;</string>
     </property>
     <property name="text">
      <string>Starting Floor</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="startingFloorInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>45</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>▲</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>▼</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ElevatorPanelFrame">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="elevatorPanel_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: Black; font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Elevator Panel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ElevatorPanel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EDE9E9
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="button5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openDoorButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>◀┃▶</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeDoorButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>▶┃◀</string>
     </property>
    </widget>
    <widget class="QPushButton" name="helpButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string>✆</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="AdminPanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>851</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EDE9E9
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="fireAlarm">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire Alarm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerOutage">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>20</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Outage</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blockDoor">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>60</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Block Door</string>
     </property>
    </widget>
    <widget class="QPushButton" name="overload">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>60</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload</string>
     </property>
    </widget>
    <widget class="QLabel" name="elevator1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator1</string>
     </property>
    </widget>
    <widget class="QLabel" name="elevator2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator2</string>
     </property>
    </widget>
    <widget class="QLabel" name="elevator3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator3</string>
     </property>
    </widget>
    <widget class="Line" name="line2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>0</y>
       <width>20</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="E1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="E2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: center;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="E3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="E1_label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>46</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: center;</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="E2_label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>45</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: center;</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="E3_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>46</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: center;</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="AdminPanelFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="AdminPanel_label">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: Black; font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Admin Panel</string>
     </property>
    </widget>
    <widget class="QLabel" name="AdminPanel_label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: Black; font-weight:bold;</string>
     </property>
     <property name="text">
      <string>System Display</string>
     </property>
    </widget>
    <widget class="Line" name="line1">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>0</y>
       <width>3</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ConsoleLogFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
font-weight: bold;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ConsoleLog_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: Black;
 font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Console Log</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="ConsoleLog_scroll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>511</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EDE9E9
</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>509</width>
       <height>329</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuElevator_Simulation">
    <property name="title">
     <string>Elevator Simulation</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBy_Vishrutha_Gopa">
    <property name="title">
     <string>By: Vishrutha Gopa</string>
    </property>
   </widget>
   <addaction name="menuElevator_Simulation"/>
   <addaction name="menuBy_Vishrutha_Gopa"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
